client/packages/invoices/src/Prescriptions/DetailView/Toolbar.tsx

```diff
@@ -30,6 +30,11 @@ export const Toolbar: FC = () => {
   const [clinicianValue, setClinicianValue] = useState<Clinician | null>(
     clinician ?? null
   );
+  const [dateValue, setDateValue] = useState(
+    DateUtils.getDateOrNull(prescriptionDate) ??
+      DateUtils.getDateOrNull(createdDatetime) ??
+      null
```

Comment:
This is the same as the previous `defaultPrescriptionDate` value, which is no longer needed (deleted below)